remote-dependency-templates/exocom/README.md

```diff
@@ -0,0 +1,47 @@
+# Exocom dependency
+
+_Production plugin for the Exocom dependency_
+
+`type: exocom`
+
+#### Template configuration in application:
+The following fields should be populated by the user:
+- `version`: Define which version of Exocom to use
+- `EXOCOM_HOST`: endpoint at which Exocom can be reached. Must be set to `exocom.<app-name>.local` as a global env var
+
+Example:
+```yml
+# application.yml
+remote:
+  dependencies:
+    <dependency-id>:
+      type: exocom
+      template-config:
+        version: 0.27.0
+  environment:
+    EXOCOM_HOST: exocom.<app-name>.local
+```
```

Comment:
I think we should move environment variables into their own own section. Same for RDS (can handle environment and secrets)